README.md

```diff
@@ -32,4 +32,7 @@ If no basic authentication is desired, make sure `Staticfile.auth` is not presen
 1. Deploy: `cf push`
 
 ## Contributors / Acknowledgments
-The first version of this public guide was authored by Jenn Noinaj and Kat Jurick and is based on a previously written guide internal to USDS. We would like to thank the following folks for their feedback, edits and support: Kathy Pham, Mark Lerner, David Nesting, Chris Given, Hank Knaack, Shannon Sartin, Eddie Hartwig, Jeff Barrett, Shelly Smith, Drew Gardner and Lisa Chung from USDS, as well as Melissa Braxton and Julie Strothman from GSA for their timely advice. Thank you!
+The first version of this public guide was authored by Jenn Noinaj and Kat Jurick and is based on a previously written guide internal to USDS, written in collaboration with GSA by co-editors Kara DeFrias and Kathy Pham with contributions from Eric Benson, Matt Collier, Evan Cook, Lisa Gelobter, Ginny Hunt, Brian Lefler, Jeff Maher, Erie Meyer, David Nesting, Chuck Rossi, Kim Rachmeler, Mollie Ruskin, Jess Teal, Albert Wong, and Charles Worthington.
+
+We would like to thank the following folks for their feedback, edits and support: Kathy Pham, Mark Lerner, David Nesting, Chris Given, Hank Knaack, Shannon Sartin, Eddie Hartwig, Jeff Barrett, Shelly Smith, Drew Gardner and Lisa Chung from USDS, as well as Melissa Braxton and Julie Strothman from GSA for their timely advice. Thank you!
```

Comment:
```suggestion
We would like to thank the following folks for their feedback, edits and support: Kathy Pham, Mark Lerner, David Nesting, Chris Given, Hank Knaack, Shannon Sartin West, Eddie Hartwig, Rachel Sauter, Jeff Barrett, Shelly Smith, Drew Gardner and Lisa Chung from USDS, as well as Melissa Braxton and Julie Strothman from GSA for their timely advice. Thank you!
```
adjustments to match the text in the most recent version of [glossary.md](https://github.com/usds/discovery-sprint-guide/blob/a890cb3d418d136335eb5b5d7209ba8791066cf6/_pages/glossary.md#acknowledgments).